src/utils/date-utils.js

```diff
@@ -45,3 +45,22 @@ export function convertPotentialTimestampToSeconds(timestamp) {
   }
   return timestampInSeconds;
 }
+
+/**
```

Comment:
Could you add a description that mentions why this method accepts three kinds of timestamps? (seconds, milliseconds, microseconds)